packages/components/src/components/Heading/Heading.tsx

```diff
@@ -6,7 +6,7 @@ import forEach from "lodash/forEach";
 import { Text } from "../../primitives/Text";
 
 type HeadingLevelOptions = "h1" | "h2" | "h3" | "h4" | "h5" | "h6";
-type HeadingMarginOptions = "space0" | "space70";
+type HeadingMarginOptions = SystemProp<keyof Theme["space"], Theme>;
```

Comment:
Why did you change this?

Reply:
well, because instead of `"space0" | "space60" | "space160" | ...` I think it's more appropriate to make this more flexible and able to receive any `spaceSuffix` available

Reply:
I understand the reason to change this, but, IMHO, it shouldn't be part of this PR. This involves changing the Heading component that's not even being mentioned in the changeset, for example.

Reply:
yeah, you're right. reverted those changes